tests/services/test_service.py

```diff
@@ -70,3 +71,99 @@ def test_simple_flow(app, consumer, service, identity_simple, input_data):
     # - search
     res = service.search(identity_simple, q=f"id:{id_}", size=25, page=1)
     assert res.total == 0
+
```

Comment:
Perhaps a fixture like this would work to avoid having to call ``current_cache.clear()`` at both the start and end of the tests.

```python
@pytest.fixture()
def cache():
    try:
        yield current_cache
    finally:
        current_cache.clear()
```

Reply:
Yeah... wanted to do that in `pytests-invenio`.... but I'll do it here and move there later...